anitya/lib/model.py

```diff
@@ -399,7 +399,7 @@ def all(cls, session, page=None, count=False):
                 page = None
 
         if page:
-            limit = page * 50
+            limit = 50
             offset = (page - 1) * 50
```

Comment:
might as well make this ``offset = (page - 1) * limit`` to make it future-proof.